1_clusters/clusters.tf

```diff
@@ -94,4 +97,35 @@ module "anthos-platform-prod-east" {
   subnetwork        = data.google_compute_subnetwork.anthos-platform-east1.name
   ip_range_pods     = "anthos-platform-pods-prod"
   ip_range_services = "anthos-platform-services-prod"
+  release_channel   = "STABLE"
 }
+
+resource "google_service_account" "gke_hub_sa" {
```

Comment:
We could probably put this in the hub module?

Reply:
i had it there, but i didn't want it repeating for each cluster, so there would be more logic needed if in hub or a variable, create new SA boolean